install/update_92_93.php

```diff
@@ -294,51 +294,64 @@ function update92to93() {
 
    if (!$DB->tableExists('glpi_racks')) {
       $query = "CREATE TABLE `glpi_racks` (
-                  `id` int(11) NOT NULL AUTO_INCREMENT,
-                  `name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
-                  `comment` text COLLATE utf8_unicode_ci,
-                  `entities_id` int(11) NOT NULL DEFAULT '0',
-                  `is_recursive` tinyint(1) NOT NULL DEFAULT '0',
-                  `locations_id` int(11) NOT NULL DEFAULT '0',
-                  `serial` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
-                  `otherserial` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
-                  `rackmodels_id` int(11) DEFAULT NULL,
-                  `manufacturers_id` int(11) NOT NULL DEFAULT '0',
-                  `racktypes_id` int(11) NOT NULL DEFAULT '0',
-                  `states_id` int(11) NOT NULL DEFAULT '0',
-                  `users_id_tech` int(11) NOT NULL DEFAULT '0',
-                  `groups_id_tech` int(11) NOT NULL DEFAULT '0',
-                  `width` int(11) DEFAULT NULL,
-                  `height` int(11) DEFAULT NULL,
-                  `depth` int(11) DEFAULT NULL,
-                  `number_units` int(11) DEFAULT '0',
-                  `is_template` tinyint(1) NOT NULL DEFAULT '0',
-                  `template_name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
-                  `is_deleted` tinyint(1) NOT NULL DEFAULT '0',
-                  `dcrooms_id` int(11) NOT NULL DEFAULT '0',
-                  `room_orientation` int(11) NOT NULL DEFAULT '0',
-                  `position` varchar(50),
-                  `bgcolor` varchar(7) DEFAULT NULL,
-                  `max_power` int(11) NOT NULL DEFAULT '0',
-                  `mesured_power` int(11) NOT NULL DEFAULT '0',
-                  `max_weight` int(11) NOT NULL DEFAULT '0',
-                  `date_mod` datetime DEFAULT NULL,
-                  `date_creation` datetime DEFAULT NULL,
-                  PRIMARY KEY (`id`),
-                  KEY `entities_id` (`entities_id`),
-                  KEY `is_recursive` (`is_recursive`),
-                  KEY `locations_id` (`locations_id`),
-                  KEY `rackmodels_id` (`rackmodels_id`),
-                  KEY `manufacturers_id` (`manufacturers_id`),
-                  KEY `racktypes_id` (`racktypes_id`),
-                  KEY `states_id` (`states_id`),
-                  KEY `users_id_tech` (`users_id_tech`),
-                  KEY `group_id_tech` (`groups_id_tech`),
-                  KEY `is_template` (`is_template`),
-                  KEY `is_deleted` (`is_deleted`),
-                  KEY `dcrooms_id` (`dcrooms_id`)
-                  ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci";
+                 `id` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
For rack, enclosures and pdu tables, we have to take care of migration from an existing master